hackathon_site/dashboard/frontend/src/components/cart/CartCard/CartCard.test.tsx

```diff
@@ -3,11 +3,12 @@ import CartCard from "components/cart/CartCard/CartCard";
 
 import { render, makeStoreWithEntities, waitFor } from "testing/utils";
 import { mockHardware } from "testing/mockData";
+import { addToCart, cartSelectors } from "slices/hardware/cartSlice";
+import { fireEvent, getByRole } from "@testing-library/react";
```

Comment:
minor: can you import `fireEvent `and `getByRole `from `testing/utils` (on line 4)